cli/azd/pkg/tools/powershell/powershell.go

```diff
@@ -25,9 +26,21 @@ type powershellScript struct {
 	envVars       []string
 }
 
+func checkInstalled(options tools.ExecOptions) (err error) {
+	return tools.ToolInPath(options.UserPwsh)
+}
+
 // Executes the specified powershell script
 // When interactive is true will attach to stdin, stdout & stderr
 func (bs *powershellScript) Execute(ctx context.Context, path string, options tools.ExecOptions) (exec.RunResult, error) {
+	if err := checkInstalled(options); err != nil {
+		return exec.RunResult{}, &internal.ErrorWithSuggestion{
+			Err: err,
+			Suggestion: "PowerShell is not installed or not in the path. To install PowerShell, " +
```

Comment:
Any thoughts if we should mention this is PowerShell **7** specifically?

Reply:
Agreed - we should add that it is Powershell 7.